woocommerce/woocommerce-android
# [Activate] Remove the button swap experiment

This PR removes the button swap experiment after it's been concluded that the swapped button order performs better.

![image](https://user-images.githubusercontent.com/1522856/193646317-f84871e8-2815-4037-b8d1-60550592f7ee.png)

**To test:**
1. Clear the data and start the app
2. Skip the prologue
3. Verify that the `Continue with WordPress.com` button is the first one
4. Rotate the screen and verify it's still the first one
